common/beerocks/bwl/dwpal/mon_wlan_hal_dwpal.cpp

```diff
@@ -361,8 +361,8 @@ static bool translate_nl_data_to_bwl_results(sChannelScanResults &results,
                                              const struct nlattr **bss)
 {
     //get bssid
-    std::copy_n(results.bssid.oct, sizeof(results.bssid.oct),
```

Comment:
Commit message typo - "bssid was wrongly copyied to results buffer." --> "bssid was wrongly copied to results buffer."